.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+s	sesudo: false
```

Comment:
Could you explain what this does? Looks like an accidental commit to me.

Reply:
Yes it was a mistake, already fixed